code/modules/projectiles/projectile/ego_bullets/zayin.dm

```diff
@@ -13,3 +13,20 @@
 	name = "9mm bullet"
 	damage = 4
 	damage_type = RED_DAMAGE
+
+/obj/projectile/ego_bullet/ego_clerk/on_hit(atom/target, blocked = FALSE)
```

Comment:
This isn't the place to do this, there's a proc(?) that checks for suicide attempt with a gun already.